src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -943,13 +941,13 @@ Now your Slack integration is available to all projects in your Sentry organizat
 
     [{% asset slack-alert-rules.png %}]({% asset slack-alert-rules.png @path %})
 
-    For any existing Alert Rules, you can edit the rule to update the Slack channel routing. When you click **Edit Rule**, under **Take these actions** you can specify your Slack channel(s):
+    For any existing Alert Rules, you can edit the rule to update the Slack channel routing. In an Alert Rule under **Perform these actions** you can pick Slack as the action:
```

Comment:
Suggest: "Update the Slack channel routing by for existing Alert Rules by choosing Slack as the action under **Perform these actions**."